src/components/layout/Navigation.spec.js

```diff
@@ -29,29 +29,47 @@ describe("isOpen state testing", () => {
 });
 
 describe("Routes should point to correct route", () => {
-  it("should point to /", () => {
+  it("clicking on about button sends user to /", async () => {
```

Comment:
All these descriptions say “about button”, but they should probably say “home”, “contact”, etc. That is, line 32, 60, and 82